src/components/Chat/ChatContent/Message/MessageContent.tsx

```diff
@@ -138,10 +138,10 @@ const ContentView = React.memo(
           <ReactMarkdown
             remarkPlugins={[
               remarkGfm,
-              [remarkMath, { singleDollarTextMath: false }],
+              remarkMath,
```

Comment:
With the disabling, it will not render variable references like for example, let $x, $y > 0.
**That said, let us prefer disable the option**.
We will see after how to instruct better behaviour.